packages/fiber/src/core/loop.ts

```diff
@@ -51,11 +49,11 @@ export function createLoop<TCanvas>(roots: Map<TCanvas, Root>) {
     })
 
     // Run global after-effects
-    for (i = 0; i < globalAfterEffects.length; i++) globalAfterEffects[i](timestamp)
+    for (let i = 0; i < globalAfterEffects.length; i++) globalAfterEffects[i](timestamp)
```

Comment:
There was a discussion on this sort of thing in the discord channel in regards to something similar in the `applyProps` function. We'd like to keep the definition of `i` above as it was. Unless you have a strong reason why we shouldn't?

Reply:
No, I don't have a 'strong reason' for this change, just thought it would be cleaner. I'll revert it.